panic: runtime error: slice bounds out of range [:1] with capacity 0

After upgrading library to the latest, getting a panic in `reader.Read()`.   This occurs with both `parquet.NewGenericReader` and `parquet.NewReader`.  Unfortunately I don't have a sanitized file that reproduces this that I can share.

```
$ go get "github.com/segmentio/parquet-go"                        
go: upgraded github.com/segmentio/parquet-go v0.0.0-20221209072905-710d87d50dd1 => v0.0.0-20221214174709-7a0ad59e0540
```

```
panic: runtime error: slice bounds out of range [:1] with capacity 0

goroutine 29 [running]:
github.com/segmentio/parquet-go.reconstructFuncOfRepeated.func1({0x1a09de0?, 0xc000330c20?, 0xc000330c00?}, {0x5e?, 0x0, 0xe?}, {0xc0003da030, 0x1d, 0x2?})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/row.go:641 +0x345
github.com/segmentio/parquet-go.reconstructFuncOfGroup.func1({0x1ab00c0?, 0xc000330c00?, 0xc00051b588?}, {0x97?, 0xf6?, 0x8f?}, {0xc0003da000, 0xc0000a9560?, 0x1f})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/row.go:788 +0x575
github.com/segmentio/parquet-go.reconstructFuncOfRepeated.func1({0x1a09da0?, 0xc0000a95d8?, 0xc0000a9560?}, {0x5e?, 0x0, 0xe?}, {0xc00026ef18, 0x1f, 0x11?})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/row.go:644 +0x203
github.com/segmentio/parquet-go.reconstructFuncOfGroup.func1({0x1aaff80?, 0xc0000a9560?, 0xc00051b7e0?}, {0x97?, 0xf6?, 0x8f?}, {0xc00026ed80, 0xc0004bc000?, 0x30})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/row.go:788 +0x575
github.com/segmentio/parquet-go.reconstructFuncOfRepeated.func1({0x1a09d60?, 0xc0004bc018?, 0xc0004bc000?}, {0x58?, 0x0, 0x51?}, {0xc0000c6b18, 0x30, 0x0?})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/row.go:644 +0x203
github.com/segmentio/parquet-go.reconstructFuncOfGroup.func1({0x1b3a140?, 0xc0004bc000?, 0xaa?}, {0x30?, 0xba?, 0x51?}, {0xc0000c6b00, 0x36?, 0x37})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/row.go:788 +0x575
github.com/segmentio/parquet-go.(*Schema).Reconstruct(0xc000048f60, {0x19f3fe0?, 0xc0004bc000?}, {0xc00031f000, 0xa6, 0xaa})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/schema.go:272 +0x29d
github.com/segmentio/parquet-go.(*Reader).Read(0xc0000a8510, {0x19f3fe0, 0xc0004bc000})
	/Users/marty/src/tempo/vendor/github.com/segmentio/parquet-go/reader.go:212 +0x246
github.com/grafana/tempo/tempodb/encoding/vparquet.findTraceByID({0x1d1f160, 0xc0001ec570}, {0xc000044280, 0x10, 0x20}, 0xc0003241a0, 0xc000164240)
	/Users/marty/src/tempo/tempodb/encoding/vparquet/block_findtracebyid.go:197 +0x908
github.com/grafana/tempo/tempodb/encoding/vparquet.(*backendBlock).FindTraceByID(0xc0002581e0, {0x1d1f0f0, 0xc00012a010}, {0xc000044280, 0x10, 0x20}, {0x400000, 0x0, 0x0, 0x0, ...})
	/Users/marty/src/tempo/tempodb/encoding/vparquet/block_findtracebyid.go:81 +0x4a5
```

**Context**
We were originally investigating a different but possibly related error on the previous version `20221209072905-710d87d50dd1`.   When reading the same row we received:
```
rs → ils → Spans → Links → no values found in parquet row for column 44
```
but after the upgrade it yields a panic.

--- a/row.go
+++ b/row.go
@@ -635,8 +635,11 @@ func reconstructFuncOfRepeated(columnIndex int16, node Node) (int16, reconstruct
 		for i := 0; i < n; i++ {
 			for j, column := range values {
 				column = column[:cap(column)]
-				k := 1
+				if len(column) == 0 {
+					continue
+				}
 
+				k := 1
 				for k < len(column) && column[k].repetitionLevel > levels.repetitionDepth {
 					k++
 				}

